ADOPTERS.md

```diff
@@ -38,15 +38,15 @@ Below is a list of solutions where Velero is being used as a component.
 We have integrated our [solution with Velero][11] to provide our customers with out of box backup/DR.
 
 **[Kyma][40]**  
-Kyma [integrates with Velero][41] to effortlessly back up and restore Kyma clusters with all its resources. Velero capabilities allow Kyma users to define and run manual and scheduled backups in order to successfully handle a disaster-recovery scenario.
+Kyma [integrates with Velero][41] to effortlessly backup and restore Kyma clusters with all its resources. Velero capabilities allow Kyma users to define and run manual and scheduled backups in order to successfully handle a disaster-recovery scenario.
```

Comment:
Kyma also seems to use "back up" on their own site, so we should probably stick with their wording here.

Reply:
Also, "backup" is a noun referring to the product of backing up. As a verb, you'd probably want to stick with "back up" -- i.e. "to back up a namespace".